jenkins/scripts/files/run_integration_tests.sh

```diff
@@ -30,7 +30,7 @@ then
   export KUBERNETES_VERSION="v1.23.8"
 fi
 
-if [ "${NUM_NODES}" == "null" ]
+if [ -z "${NUM_NODES+set}" ] | [ "${NUM_NODES}" == "null" ]; 
```

Comment:
There should be `||` (or) instead of `|` (pipe), and I don’t follow what you’re trying to do with `+set`?

Reply:
my bad, Fixed it.